### ISSUE TITLE
is this the code for your paper of "Exposure vs Contrast Enhancement in low light 4K videos", published in 2020?

### ISSUE THREAD

**Issue by Dongshengjiang** [sohaibali01/low-light-video-enhancement#2]
I am confused the code is release in three years ago. but your paper was published recently?

**Comment by sohaibali01** [comment]
yes, rejection from couple of journals before accepted here, https://link.springer.com/article/10.1007/s11760-020-01714-3. I was also lazy during this whole episode of resubmissions :)
